app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/address/Address.scala

```diff
@@ -31,7 +31,7 @@ object Address {
 
   val addressLineAllowedCharacters: List[Char] =
     ('A' to 'Z').toList ::: ('a' to 'z').toList ::: ('0' to '9').toList :::
-      List(' ', '-', ',', '.', '&', '\'')
+      List(' ', '-', ',', '.', '&', '\'', '/')
 
```

Comment:
Please test it throughly with unit tests for all controllers where we use this logic